src/parser/uniterm.rs

```diff
@@ -211,7 +211,7 @@ impl UniRecord {
                                         types: Some(ctrt),
                                         contracts,
                                         opt: false,
-                                        priority: MergePriority::Normal,
+                                        priority: MergePriority::Neutral,
```

Comment:
An alternative could be to use `Option<float>`.
then `Bottom` become `Some(-float::inf())`, `Top` become `Some(float::inf())`, and `Neutral` become `None`
Sorry not sure how to write infinite in Rust but that's the idea.
May easy the priority comparaisons but may be less clear in the code...